brave-spancollector-scribe/src/test/java/com/github/kristofa/brave/scribe/CountingScribeSpanCollectorNotificationsHandler.java

```diff
@@ -0,0 +1,53 @@
+package com.github.kristofa.brave.scribe;
+
+import com.twitter.zipkin.gen.Span;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+class CountingScribeSpanCollectorNotificationsHandler implements ScribeSpanCollectorNotificationsHandler {
+
+    private final AtomicInteger spansRejected = new AtomicInteger();
+    private final AtomicInteger spansSentEvents = new AtomicInteger();
+    private final AtomicInteger sendingFailures = new AtomicInteger();
+    private volatile int lastQueueSize;
```

Comment:
You're right, but performance was not my concern here, it's just for checking the notifications logic in the unit tests. This class has no other intention.